src/processors/encounter/encounter-form-processor.ts

```diff
@@ -175,7 +177,7 @@ export class EncounterFormProcessor extends FormProcessor {
       // handle diagnoses
       if (savedDiagnoses.length) {
         showSnackbar({
-          title: translateFn('diagnosisSaved', 'Diagnosis(es) saved successfully'),
+          title: t('diagnosisSaved', 'Diagnosis(es) saved successfully'),
```

Comment:
```suggestion
          title: t('diagnosisSaved', 'Diagnosis saved successfully', { count: savedDiagnoses.length }),
```

This passes the count parameter to i18next, enabling it to use the appropriate plural form (one/_other suffix) based on the number of diagnoses. If this doesn't work, we should default to the plural "Diagnoses saved successfully" to avoid localization issues with parentheses.

Reply:
@denniskigen I think this can be addressed in a separate PR. Gonna have this merged in for now.